<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sorular</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblSoru">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>641</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnA">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnB">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>160</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnC">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnD">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>240</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Soru Sayısı :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSoruSayisi">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Puan :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPuan">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl500">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>10</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>500</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl1000">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>40</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1.000</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl2000">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>70</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>2.000</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl3000">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>100</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>3.000</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl5000">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>130</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>5.000</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl7500">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>160</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>7.500</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl15000">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>190</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>15.000</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl30000">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>220</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>30.000</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl60000">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>250</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>60.000</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl125000">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>280</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>125.000</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl250000">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>310</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>250.000</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl1000000">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>340</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1.000.000</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSure">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>962</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
